source/_patterns/01-molecules/components/section-listing.mustache

```diff
@@ -2,7 +2,7 @@
   {{#listHeading}}
     {{> atoms-list-heading}}
   {{/listHeading}}
-  <ol class="section-listing {{#singleLine}}section-listing--single-line{{/singleLine}}" {{#labelledBy}}aria-labelledby="{{labelledBy}}"{{/labelledBy}}>
```

Comment:
What was the reason for removing the labelledBy aria tag

Reply:
The reason I deleted that line was that it was no longer necessary. I asked before if it is okay to delete what is no longer necessary.